ProcessesApi/V1/Services/SoleToJointService.cs

```diff
@@ -131,6 +131,33 @@ private async Task CheckTenureInvestigation(StateMachine<string, string>.Transit
             await TriggerStateMachine(processRequest).ConfigureAwait(false);
         }
 
+        private async Task CheckHOApproval(StateMachine<string, string>.Transition transition)
+        {
+            var processRequest = transition.Parameters[0] as ProcessTrigger;
+            var formData = processRequest.FormData;
+
+            var expectedFormDataKeys = new List<string> { SoleToJointFormDataKeys.HORecommendation };
+            SoleToJointHelpers.ValidateFormData(formData, expectedFormDataKeys);
+            var housingOfficerRecommendation = formData[SoleToJointFormDataKeys.HORecommendation].ToString();
+
+            switch (housingOfficerRecommendation)
```

Comment:
We could maybe have a helper method that evaluates form data values and sets the trigger based on the value (using params), similar to the ValidateManualChecks method